packages/SketchMorph2-Core.package/M2Backend.class/instance/saveSettingsDuring..st

```diff
@@ -1,10 +1,14 @@
 undo
 saveSettingsDuring: aBlock
-	| preColor preFillingGeometric preTool |
+	| preBrushSize preColor preFillingGeometric preTool |
+	preBrushSize := self brushSize.
 	preColor := self currentColor.
 	preFillingGeometric := self isFillingGeometricForm.
 	preTool := self currentTool.
 	
 	aBlock value.
-	self restoreColor: preColor isFilling: preFillingGeometric tool: preTool.
+	self restoreBrushSize: preBrushSize
+		color: preColor
+		isFilling: preFillingGeometric
+		tool: preTool.
 	self reloadCurrentPen
```

Comment:
Actually, I meant something like that
```suggestion
	self
		restoreBrushSize: preBrushSize
		color: preColor
		isFilling: preFillingGeometric
		tool: preTool;
		reloadCurrentPen
```

Reply:
But this isn't what is style is meant to be, as far as I know. In this style, the difference between the method call to `restoreBrushAndEraserSize:color:isFilling:tool:` and to `reloadCurrentPen` isn't that obvious.